sys/reaper/reaper_darwin.go

```diff
@@ -0,0 +1,60 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package reaper
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/containerd/containerd/pkg/process"
+	runc "github.com/containerd/go-runc"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// reapMore is additional reap process upon receipt of SIGCHLD.
+// Since macOS doesn't raise SIGCHLD on orphaned children's exit,
+// reapMore polls the status of registered process and terminate it
+// if it's already exited.
+
+func reapMore(bundlePath string, exits []exit) ([]exit, error) {
+	pid, err := runc.ReadPidFile(filepath.Join(bundlePath, process.InitPidFile))
```

Comment:
`bundlePath` can be `""`

```suggestion
func reapMore(bundlePath string, exits []exit) ([]exit, error) {
	if bundlePath == "" {
		return nil, nil
	}
	pid, err := runc.ReadPidFile(filepath.Join(bundlePath, process.InitPidFile))
```

Reply:
I investigated this again and found that the parameter `bundlePath` is not needed to pass (it can be estimated from the current directory) so, reverted the related changed to the original one.